src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -233,8 +233,7 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
     args: {
       selection: {
         inputType: 'selection',
-        // TODO: These are products of an extrude
-        selectionTypes: ['extrude-wall', 'start-cap', 'end-cap'],
+        selectionTypes: ['solid2D', 'segment'],
```

Comment:
I think this is of interest to some folks @franknoirot @nadr0 @max-mrgrsk 

This and some of the command bar selections logic is somewhat fixed, I'll leave a few other comments of things I wasn't sure about.